tests/verification.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from typing import Any
+
+# create and configure main logger
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+# create console handler with a higher log level
+LOG_HANDLER = logging.StreamHandler()
+LOG_HANDLER.setLevel(logging.DEBUG)
+
+# create formatter and add it to the handler
+FORMATTER = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+LOG_HANDLER.setFormatter(FORMATTER)
+
+# add the handler to the logger
+logging.getLogger('').addHandler(LOG_HANDLER)
+
+
+class Verifier:
+    """
+
+    """
+    def __init__(self, name: str = "Verifier") -> None:
+        self.logger = LOGGER
+        self.logger.info(name)
+
+    def fail(self, message: str = "") -> None:
+        """
+        Fail the test by raising an assertion error
+
+        Parameters:
+            message:
+                Message to be logged.
+
+        Raises:
+            AssertionError:
+                An error occurred.
+
+        """
+        raise AssertionError(f"TEST FAILURE: {message}")
+
+    def verify_equals(self, actual: Any, expected: Any,
```

Comment:
suggestion: It would better to write [`__eq__`](https://docs.python.org/3/library/operator.html#operator.eq) operator here rather than writing a regular method. Then the objects extended for this capability would be easy to use and could extend specific features to be compared to the sub-classes if required.

Reply:
Thanks for the recommendation.  I hadn't used this operator module before.  Let me know if the change is what you were thinking.